lib/tasks/integration.rake

```diff
@@ -0,0 +1,96 @@
+INTEGRATION_TMP_DIR     = Rails.root.join('tmp', 'integration')
```

Comment:
Will Rails constant always exist at this time (even if we don't have the environment, like when calling `rake -T`?)

Reply:
Good catch, though I think with how our `Rakefile` is:

https://github.com/ManageIQ/manageiq/blob/c157a8130c47220b830e0a72ffadcebcd2f78ed4/Rakefile#L5

This will always be available.  That isn't to say I am against doing this in a different way (and I did attempt making this unavailable for [certain rake a few years ago](https://github.com/ManageIQ/manageiq/pull/14916) for performance reasons), just that it should work for now.

Reply:
Going to resolve this for now, though, if you want to re-open it so it is on our radar as something to change in the future, go ahead.

(just trying to keep track of what I have already addressed)